crates/csharp/Cargo.toml

```diff
@@ -1,7 +1,7 @@
 [package]
 name = "wit-bindgen-csharp"
 authors = ["Timmy Silesmo <silesmo@nor2.io>"]
-version = "0.16.0"
+version = "0.17.0"
```

Comment:
I am not sure how ready Csharp bindgen is - could we add some documentation on the README about it? @jsturtevant @silesmo